src/main/resources/META-INF/jpa-schema-export.sql

```diff
@@ -396,18 +385,29 @@
         primary key (id)
     ) engine=MyISAM;
 
-    create table LetterSoundCorrespondence_Letter (
-       LetterSoundCorrespondence_id bigint not null,
+    create table LetterSound_Letter (
+       LetterSound_id bigint not null,
         letters_id bigint not null,
         letters_ORDER integer not null,
-        primary key (LetterSoundCorrespondence_id, letters_ORDER)
+        primary key (LetterSound_id, letters_ORDER)
     ) engine=MyISAM;
```

Comment:
@SnehaHS65 FYI, here is the current structure of the `LetterSoundCorrespondence_Letter` table in the production database:

```sql
MariaDB [webapp-HIN]> DESCRIBE LetterSoundCorrespondence_Letter;
+------------------------------+------------+------+-----+---------+-------+
| Field                        | Type       | Null | Key | Default | Extra |
+------------------------------+------------+------+-----+---------+-------+
| LetterSoundCorrespondence_id | bigint(20) | NO   | PRI | NULL    |       |
| letters_id                   | bigint(20) | NO   | MUL | NULL    |       |
| letters_ORDER                | int(11)    | NO   | PRI | NULL    |       |
+------------------------------+------------+------+-----+---------+-------+
```

```sql
MariaDB [webapp-HIN]> SHOW CREATE TABLE LetterSoundCorrespondence_Letter;
+----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table                            | Create Table

                                                                                                                           |
+----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| LetterSoundCorrespondence_Letter | CREATE TABLE `LetterSoundCorrespondence_Letter` (
  `LetterSoundCorrespondence_id` bigint(20) NOT NULL,
  `letters_id` bigint(20) NOT NULL,
  `letters_ORDER` int(11) NOT NULL,
  PRIMARY KEY (`LetterSoundCorrespondence_id`,`letters_ORDER`),
  KEY `FK_f3mr6ddn5ctr8edvv9pgtjh2j` (`letters_id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
+----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
@jo-elimu , Yes added for both LetterSoundCorrespondence_Letter and LetterSoundCorrespondence_Sound in the same db migration script.